src/nbt/tag.rs

```diff
@@ -74,28 +77,21 @@ impl NbtTag {
         bytes.freeze()
     }
 
-    /// Serializes as single NBT tag without name.
-    pub fn serialize_tag(&self) -> Bytes {
+    /// Writes tag id and its data into bytes
+    pub fn serialize(&self) -> Bytes {
         let mut bytes = BytesMut::new();
         bytes.put_u8(self.id());
         bytes.put(self.serialize_raw());
         bytes.freeze()
     }
 
-    /// Serializes the NBT tag into bytes with a name and id.
-    pub fn serialize_named(&self, name: &str) -> Bytes {
```

Comment:
I don't understand why you removed it. This method is used to serialize normal tag (the same way as in NBT). This can be useful. Remember, we are writing library, so why don't provide such methods?

Reply:
What use do you see for this?
Note that this method can't be used to serialize data into for example a file, because the root tag needs to be a compound, while this method allows an arbitary type like a list, string, etc, so it can produce invalid nbt this way.

Reply:
In Minecraft, Mojang sometimes uses random methods, like using unnamed single tags for text components. It's better to keep these methods available, even if they seem unused, to avoid others having to create new NBT libraries lol.

Reply:
> In Minecraft, Mojang sometimes uses random methods, like using unnamed single tags for text components. It's better to keep these methods available, even if they seem unused, to avoid others having to create new NBT libraries lol.

Well, we are talking about named tag, which is only used in compound. Unnamed one makes sense, because that's how type are encoded everywhere else - id and then data, and because of that we have serialize.
I don't see any good reason to keep it.